README.md

```diff
@@ -34,78 +64,99 @@ re.test('👨🏾‍⚕️'); // '\u{1F468}\u{1F3FE}\u200D\u2695\uFE0F'
 
-<pre>\p{<b><i>UnicodeSequencePropertyName</i></b>}</pre>
+<pre>\p{<b><i>PropertyName</i></b>}</pre>
 
-The negated `\P{…}` form is not supported for sequence properties as it would be [a footgun](https://github.com/tc39/proposal-regexp-unicode-sequence-properties/issues/6#issuecomment-368460069). It’s not generally useful, and is better expressed as a negative lookahead. Compare the unsupported `/\P{UnicodeSequenceProperty}/u` (what should it do?) with `/(?!\p{UnicodeSequenceProperty})/u` (clear what it does).
+Where `PropertyName` can be one of the properties of strings listed above.
 
-Given that `UnicodeSequencePropertyName` expands to a list of sequences of Unicode code points, the proposal also includes a static restriction that bans such properties within character classes.
+The complement of such a property is not supported:
+Both `\P{PropertyName}` and `[^...\p{PropertyName}...]` will fail to compile (throw a `SyntaxError` exception)
+if `PropertyName` is a property of strings.
 
-### FAQ
-
-#### What about backwards compatibility?
+We have thought of possible definitions of such a complement, but we believe that they are not generally useful.
 
-Unicode property escapes for unsupported Unicode properties throw an early `SyntaxError`. As such, we can add support for new properties in a backwards-compatible way, as long as we re-use the existing syntax.
+Some of the use cases for “not a property of strings” can be supported via a negative lookahead:
+`/(?!\p{RGI_Emoji_Flag_Sequence})\p{Symbol}/u`
 
-#### Why ban the use of these properties within character classes?
+Note: Using a property of strings inside a character class is equivalent to
+an alternation of all of the strings and characters,
+such that the order of elements is irrelevant (e.g., listing the strings longest-first).
+(This could be optimized by retaining a character class of the single characters,
+as illustrated in the Motivation section above.)
 
-Currently, each property escape expand to a list of code points. As such, their meaning is clear and unambiguous, even within a character class. For example, the following regular expression matches either a `Letter`, a `Number`, or an underscore:
+### FAQ
 
-```js
-const re = /[\p{Letter}\p{Number}_]/u;
-```
+#### What about backwards compatibility?
 
-For the new properties introduced by this proposal, the expected behavior within character classes is unclear. A character class, when matched, always produces only a single character. Allowing sequence properties within character classes would change that, for no good reason.
+Unicode property escapes for unsupported Unicode properties throw an early `SyntaxError`. As such, we can add support for new properties in a backwards-compatible way, as long as we re-use the existing syntax.
 
-```js
-const re = /[\p{RGI_Emoji_ZWJ_Sequence}_a-z]/u;
-// 🤔 What should this do?
+#### Properties of strings within character classes
```

Comment:
This is a normative change to the proposal. We should call this out next time we present, and clarify (here in the README) that this is partially motivated by the set notation proposal.

This can be done as a follow-up patch, so I'll go ahead and merge this now.